src/Components/StudentsList/Pagination.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from "react";
+
+function Pagination() {
+  const [page, setPage] = useState(1);
```

Comment:
Te page and setPage should come form heigher component. Pagination component is only a control component so it shouldn't have it own state.